config.sample.json

```diff
@@ -28,6 +28,7 @@
     "features": { },
     "default_federate": true,
     "default_theme": "light",
+    "map_style_url": "https://api.maptiler.com/maps/basic/style.json?key=YOUR_KEY_GOES_HERE",
```

Comment:
including this in the sample config will just cause most deployments to end up with this placeholder API Key which will lead to many unexpected experiences, imo this shouldn't be in the sample config